creator-node/src/routes/tracks.js

```diff
@@ -40,20 +40,28 @@ module.exports = function (app) {
     // for each path, call saveFile and get back multihash; return multihash + segment duration
     // run all async ops in parallel as they are not independent
     const saveSegmentFileTimeStart = Date.now()
-    let saveFileProms = []
-    let durationProms = []
     const t = await models.sequelize.transaction()
-    for (let filePath of segmentFilePaths) {
+
+    req.logger.info(`segmentFilePaths.length ${segmentFilePaths.length}`)
```

Comment:
@SidSethi do you want to remove these logs? they slow down the processing significantly

Reply:
as discussed afk, keeping for debugging now will remove soon